jetstream_pt/engine.py

```diff
@@ -147,29 +151,27 @@ def _call_model_generate(
     if self.env.enable_kv_quantization:
       caches_obj = [
           cache_manager.Int8KVCacheGenerate(k, v, ks, vs, input_indexes)
-          for (k, v), (ks, vs) in torch_xla2.tensor.wrap(
-              list(zip(caches, cache_scales))
-          )
+          for (k, v), (ks, vs) in from_jax(list(zip(caches, cache_scales)))
```

Comment:
Nit, since Pytorch should be the focus, is it better to use "to_torch" and "from_torch" instead of "from_jax" and "to_jax"? Ideally we should not see any Jax related term.

Reply:
done